packages/schemas/src/categories/category-update.dto.ts

```diff
@@ -0,0 +1,13 @@
+import { IsNotEmpty, IsIn } from "class-validator";
+import { CategoryType, CategoryTypeValues } from './category-type.js';
+
+export class CategoryUpdateDto {
+  @IsNotEmpty()
+  id: number;
+
+  @IsNotEmpty()
+  name: string;
+
+  @IsIn(CategoryTypeValues)
+  type: CategoryType;
```

Comment:
Ông nên bỏ trường `type` ở đây, do không được update type của category

Reply:
T thấy hàm `findCategoryByNameAndType` nó nhận type làm parameter nên t nghĩ vẫn cần nó
Với lại
`await this.categoriesRepo.update(dto.id, {
      name: dto.name
 });`
Chỉ update name thôi chứ hả